src/Vipps.net/Services/AccessTokenService.cs

```diff
@@ -6,26 +6,44 @@
 
 namespace Vipps.net.Services
 {
-    public static class AccessTokenService
+    public interface IVippsAccessTokenService
```

Comment:
Maybe just forgot to rename the file, the file name should be the same as the interface/class